bin/seed.js

```diff
@@ -7,6 +7,394 @@ const init = async () => {
     try {
         await db.sync({ force: true })
         await Category.create({ name: 'salty' })
+        const Puff = await Product.create({
+            title: 'Strawberry Puff',
+            brand: 'I-Mei',
+            description: 'Crispy puff shell cookie with cream filling.',
+            category: 'sweet',
+            price: 3.99,
+            inventory: 100,
+            country: 'Taiwan',
+            imageUrl:
+                'https://images-na.ssl-images-amazon.com/images/I/51GkdanTqfL.jpg',
+        })
+    
+        const PineappleCake = await Product.create({
+            title: 'Pineapple Cake',
+            brand: 'Jun-Mei',
+            description:
+                'A buttery, shortbread-like treat with a pineapple jam filling.',
+            category: 'sweet',
+            price: 36.99,
+            inventory: 139,
+            country: 'Taiwan',
+            imageUrl:
+                'https://www.food168.com.tw/upload_files/a2L-detail.jpg?fbclid=IwAR19giBuwx8ZA1xGzD6kzX2oMttS4796rIC8lLPGhcNTuoAzDHQbipL-e0c',
+        })
+    
+        const IceCreamBar = await Product.create({
+            title: 'Black Sugar Boba Ice Cream Bar',
+            brand: 'Tigersugar',
+            description:
+                'It contains chewy tapioca pearls mingle with a milk-based tea.',
+            category: 'sweet',
+            price: 4.99,
+            inventory: 126,
+            country: 'Taiwan',
+            imageUrl:
+                'https://sethlui.com/wp-content/uploads/2019/11/Tiger-Sugar-Boba-Ice-Cream-Online-2.jpg',
+        })
+    
+        const Roll = await Product.create({
+            title: 'Multi Grain Crispy Roll',
+            brand: 'Pei Tien',
+            description:
+                'Multi Grain Crispy Roll is made from 7 natural grains,not too sweet and with a hint of salt.',
+            category: ['sweet', 'salty', 'healthy'],
```

Comment:
Line 53 - not sure if you put an array as a value, for now just put one and comment somewhere the others, so when I create the association table, we can add them

Reply:
I think this happens with a couple other products as well

Reply:
In line 53, this product actually belongs to all these three categories...I will check others as well. So if the product only belongs to ONE category, do I still have to put it in an array??

Reply:
Since we are getting rid of the category field, you can delete it in the create()s you do. We can add the categories afterwards.

Reply:
Ok